proposals/selinux.md

```diff
@@ -0,0 +1,48 @@
+# OCI go-selinux project proposal
+
+## Abstract
+Having a solid, battle-proven, common selinux implementation in OCI for use in and outside of runc will ensure long lasting security and interoperability throughout the container ecosystem.
+
+## Proposal
+Refactor and move the selinux library out of runc into a separate project:
+
+https://github.com/opencontainers/runc/tree/master/libcontainer/selinux
+
+The new project would live in the opencontainers github organization:
+
+https://github.com/opencontainers/go-selinux
+
+A sample of how the project would look like is already here:
+
+https://github.com/runcom/go-selinux
+
+### Initial Maintainers
+Initial maintainers of the go-selinux project would be:
+
+* Antonio Murdaca <runcom@redhat.com> (@runcom)
+* Daniel J Walsh <dwalsh@redhat.com> (@rhatdan)
+* Mrunal Patel <mpatel@redhat.com> (@mrunalp)
+* TODO
+
+### Code of Conduct
+This project would incorporate (by reference) the OCI Code of Conduct.
+
+### Governance and Releases
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
```

Comment:
Both?  Copy/paste from somewhere else?  I think there's only one project here, unless you mean runC is not leaving.  Either way, I think this line could be more clear (e.g. listing projects by name).